scripts/bee.sh

```diff
@@ -21,6 +21,7 @@ PARAMETERS:
     --own-image                 If passed, the used Docker image names will be identical as the name of the workers.
     --version=x.y.z             used version of Bee client.
     --detach                    It will not log the output of Queen node at the end of the process.
+    --payment-tolerance         excess debt above payment threshold in BZZ where you disconnect from your peer (without decimals, default: 50000000000000)
```

Comment:
The default is not correct. Both places where it is actually defined have different value.